lake/top/lake_top.py

```diff
@@ -674,7 +698,7 @@ def __init__(self,
             kts.passes.auto_insert_sync_reset(self.internal_generator)
 
         # config regs
-        lift_config_reg(self.internal_generator)
+        # lift_config_reg(self.internal_generator)
```

Comment:
?

Reply:
This lift has been moved up to lake_chain.py, so having it here also results in lifting twice which causes issues. I can delete this line if it's confusing.